README.md

```diff
@@ -67,6 +81,22 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
 	- [Russian](#russian)
 
 
+Icons and tags:
+
+- Ancient: ![ancient]
+- Commercial: ![$]
+- Crush: ![heart]
+- Supporting: ![support]
+- Up and coming: ![upcoming]
```

Comment:
I think there is a slight distinction between the 2, Beta might not be ready for production. Up and coming is ready for production and is undergoing accelerated development/adoption.